[IMP] sales: pricelists tabs (#11918)
odoo/documentation: content/applications/sales/sales/products_prices/prices/pricing.rst

  - :guilabel:`Discount`: Enter the percentage to be discounted. A mark-up can be configured by
    using a negative value in this field.

    .. important::

>        If a pricelist is applied to a sales quotation with discounts configured under
>        :guilabel:`Price Type` > :guilabel:`Discount`, the discount will visible to the customer.

Reviewer: Please avoid using symbolism to describe UI elements :)
```suggestion
       If a pricelist is applied to a sales quotation with the discount :guilabel:`Price Type` set
       as :guilabel:`Discount`, the discount will visible to the customer.
```

>        .. image:: pricing/sales-pricelist-pricerule-discount.png

Reviewer 2: Would it be possible to redo this screenshot to **only** show the "Open: Pricelist Rules" pop-up? The way it is now, it is fairly blurry. I think if you focus on just the pop-up it will show the information you need and will be clearer.

          :alt: A pricelist rule using Price Type: Discount to create a discount.

       .. image:: pricing/sales-pricelist-pricerule-discount-customer.png
          :alt: How a pricelist rule using Price Type: Discount appears on customer preview.

  - :guilabel:`Formula`: Calculate the pricelist rules based the following configuration:

    - :guilabel:`Discount`: Percentage discount to be applied. Negative values can be entered to
      increase prices.

    .. important::

>        If a pricelist is applied to a sales quotation with discounts configured under
>        :guilabel:`Price Type` > :guilabel:`Formula`, the discount will *not* be visible to the
>        customer.

Reviewer: ```suggestion
       If a pricelist is applied to a sales quotation with the discount :guilabel:`Price Type` set
       as :guilabel:`Formula`, the discount will *not* be visible to the customer.
```

  applied to quotations.

.. example::

>    To formulate a 100% markup (or 2 times the price of the product), with a $5 minimum margin, set

Reviewer 2: ```suggestion
   To formulate a 100% markup (or two times the price of the product), with a $5 minimum margin, set
```

--------------------

Recurring prices are specifically used with :doc:`subscription products
<../../../subscriptions/products>`. When configuring price rules in this tab, keep in mind that
they will only apply to subscription products, or products with recurring prices enabled.

In the :guilabel:`Recurring Prices` tab, pricelists are configured with the same options as in the
:guilabel:`Price Rules` tab, with additional columns for :guilabel:`Product Variants` add

> ::guilabel:`Recurring Plan`.

Reviewer: Remove extra `:`
```suggestion
:guilabel:`Recurring Plan`.
```

:guilabel:`Product Variants` are configured under products that have one or more values, such as

> color, size, etc. Once a product has been selected under the :guilabel:`Products Tab`, if applicable
> , select the desired product variants to be included in the price rule.

Reviewer: Please only break a line where a ` ` space exists, otherwise a space is added due to the line break.

```suggestion
color, size, etc. Once a product has been selected under the :guilabel:`Products Tab`, if
applicable, select the desired product variants to be included in the price rule.
```
![Screenshot 2025-01-30 at 5 10 37 PM](https://github.com/user-attachments/assets/3d390c62-ebed-42ca-b12c-ec8f275e5cef)

Then, select the blank field in the :guilabel:`Recurring Plan` column to reveal a drop-down menu of
pre-designated recurrence periods (e.g. `Monthly`, `Quarterly`, `Weekly`, etc.).

.. image:: pricing/sales-pricelist-recurringprices.png
   :alt: The recurring prices tab in a pricelist configuration form.

New recurrence periods can also be created from this column. To do so, type in the name for the new
:guilabel:`Recurring Plan`, then select :guilabel:`Create` from the resulting drop-down menu to

> create the time period, which can be edited later. Alternatively, select :guilabel:
> `Create and edit...`to reveal a :guilabel:`Create Recurring Plan` pop-up form. From this pop-up form
> , the new recurrence period can be configured, with specific :guilabel:`Details`, :guilabel:
> `Self-Service`, and :guilabel:`Pricing` options. When the configurations are complete, click the
> :guilabel:`Save & Close` button.

Reviewer: Be careful breaking lines :), they can only be broken where a space is. For example:
```
YES: select :guilabel:`Create and✅edit...` to
NO:  select :guilabel:❌`Create and edit...` to
```

![Screenshot 2025-01-30 at 5 12 05 PM](https://github.com/user-attachments/assets/17b68921-f3f9-47e2-9262-d1e67e935f09)

```suggestion
create the time period, which can be edited later. Alternatively, select :guilabel:`Create and
edit...` to reveal a :guilabel:`Create Recurring Plan` pop-up form. From this pop-up form, the new
recurrence period can be configured, with specific :guilabel:`Details`, :guilabel:`Self-Service`,
and :guilabel:`Pricing` options. When the configurations are complete, click the :guilabel:`Save &
Close` button.
```

Rental rules tab
----------------

> Price rules can be configured for :doc:`rental products <../../../sales/rental>` under the

Reviewer 3: [CI error](https://runbot.odoo.com/runbot/build/74192048)
when we hop back three directories, we're still in the sales section so no need for it here.
```suggestion
Price rules can be configured for :doc:`rental products <../../../rental>` under the
```

Lastly, configure a :guilabel:`Price` for the rental rule in the respective column.

.. image:: pricing/sales-pricelist-rental.png
   :alt: The rental tab in a pricelist configuration form.

Ecommerce Tab
-------------

Under the :guilabel:`Ecommerce` tab, price rules can be configured for products sold on an

> :doc:`Ecommerce website <../../../../websites/ecommerce/products>`.

Reviewer 2: ```suggestion
:doc:`eCommerce  website <../../../../websites/ecommerce/products>`.
```
I will leave a note when I finish my review about the e-commerce capitals :)

> Under the :guilabel:`Ecommerce` tab, price rules can be configured for :doc:`products sold on an
> :doc:`Ecommerce website <../../../websites/ecommerce/products>`.

Reviewer 3: • remove duplicate `:doc:` tag
• [CI error 2](https://runbot.odoo.com/runbot/build/74192048): Needed to hop back one more time so we were fully out of the sales app directory, in order to proceed into `/websites`
```suggestion
Under the :guilabel:`Ecommerce` tab, price rules can be configured for products sold on an
:doc:`Ecommerce website <../../../../websites/ecommerce/products>`.
```

Reviewer 3: **Note**
Consider there are multiple ways eCommerce is spelled in this doc. With the exception of UI labels, we prefer the lowercase `e` version written, no hyphen.

To enable the pricelist to be visible, select the target website in the :guilabel:`Website` field.

The :guilabel:`Selectable` can be enabled to allow the customer to choose this pricelist.

> Finally, promotional and loyalty codes can e added to the :guilabel:`E-commerce Promotional Code`

Reviewer 3: 🐝 
```suggestion
Finally, promotional and loyalty codes can be added to the :guilabel:`E-commerce Promotional Code`
```

field.

.. image:: pricing/sales-pricelist-ecommerce.png

>    :alt: The ecommerce tab in a pricelist configuration form.

Reviewer 2: ```suggestion
   :alt: The Ecommerce tab in a pricelist configuration form.
```
